src/Integration/Binding/DotNetBindingConfigProvider.cs

```diff
@@ -105,7 +112,12 @@ public async Task<IBindingConfigFile> GetConfigurationAsync(SonarQubeQualityProf
             // Remove/Move/Refactor code when XML ruleset file is no longer downloaded but the proper API is used to retrieve rules
             UpdateDownloadedSonarQubeQualityProfile(ruleSet, qualityProfile, bindingConfiguration.Project.ProjectName, bindingConfiguration.Project.ServerUri.ToString());
 
-            return new DotNetBindingConfigFile(ruleSet);
+            var ruleSetFilePath = solutionBindingFilePathGenerator.Generate(
```

Comment:
Calculation of ruleSetPath for IBindingConfigFile. Same as
https://github.com/SonarSource/sonarlint-visualstudio/pull/1365/files#r418649075